content/ember-data/v3/default-serializer.md

```diff
@@ -41,8 +43,4 @@ If a specific model type requires custom serialization, a type-specific serializ
     }
 ```
 
-Defining a serializer for the entire application can be done by adding the file `app/serializers/application.js` with the following:
-
-```js
-    export { default } from '@ember-data/serializer/json-api';
-```
+More information about custom serializers can be found in the [Serializer API Docs](https://api.emberjs.com/ember-data/release/modules/@ember-data%2Fserializer) or on the [ember.js/guides](https://guides.emberjs.com/release/models/customizing-serializers/#toc_customizing-serializers)
```

Comment:
```suggestion
More information about custom serializers can be found in the [Serializer API Docs](https://api.emberjs.com/ember-data/release/modules/@ember-data%2Fserializer) or on the [official guides](https://guides.emberjs.com/release/models/customizing-serializers/#toc_customizing-serializers)
```

Reply:
English question, does anyone have a good mnemonic for "on the guides" vs "in the guides"?

Reply:
I think I prefer in, but don't have super strong opinions. Probably would make more sense to use on if were saying `on the Guides website`, but either way I think it def would sound better if we unified in/on b/w the api docs and guides here.